packages/backend/src/plugins/identity.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createRouter } from '@backstage/plugin-identity-backend';
+import { PluginEnvironment } from '../types';
+
+export default async function ({ logger }: PluginEnvironment) {
+  return await createRouter({ logger });
```

Comment:
I think freben commented on this for the auth backend, but afaik the logger for the plugin should be created here.

Reply:
@Rugvip - I moved the logger creation to this file. Can you check?

Reply:
nvm, seems like it's done as a part of setting up the env in the backend so no need to be done there